controllers/nugus_controller/nugus_controller_test.cpp

```diff
@@ -169,11 +173,14 @@ int main(int argc, char** argv) {
             current_num += 1;
             msg.set_num(current_num);
 
-            N = msg.ByteSizeLong();
-            data.resize(N);
-            msg.SerializeToArray(data.data(), N);
+            Nh = msg.ByteSizeLong();
+            data.resize(Nh);
+            msg.SerializeToArray(data.data(), Nh);
 
-            if (send(tcp_fd, &N, sizeof(N), 0) < 0) {
+            // Covert to network endianness, which might be different to network endianness
```

Comment:
You spotted my error, then added the comment anyway ahaha
```suggestion
            // Covert to network endianness, which might be different to host endianness
```